packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -696,10 +696,12 @@ export class BasicCrawler<
         if (shouldRetryRequest) {
             request.retryCount++;
             const { url, retryCount, id } = request;
-            this.log.exception(
-                error,
-                'handleRequestFunction failed, reclaiming failed request back to the list or queue',
-                { url, retryCount, id },
+            // We don't want to see the stack trace in the logs by default, when we are going to retry the request.
+            // Thus, we print the full stack trace only for DEBUG log level.
+            // Maybe it's worth adding another ENV variable instead of using log levels here.
+            this.log.warning(
+                `Reclaiming failed request back to the list or queue. ${this.log.getLevel() < this.log.LEVELS.DEBUG ? error : error.stack}`,
```

Comment:
let's use env var for this instead of log levels, maybe `CRAWLEE_VERBOSE_LOG` or something along that